packages/modules/vehicles/evcc/api.py

```diff
@@ -69,7 +69,7 @@ def fetch_soc(
         stub = vehicle_pb2_grpc.VehicleStub(channel)
 
         if not evcc_config.vehicle_id:  # create and fetch vehicle id if not included in config
-            create_and_save_vehicle_id(stub, evcc_config, vehicle)
+            vehicle_to_fetch = create_and_save_vehicle_id(stub, evcc_config, vehicle)
 #            vehicle_to_fetch = create_vehicle(evcc_config, stub)
 #            log.debug("Vehicle client received: " + str(vehicle_to_fetch))
```

Comment:
```suggestion
            vehicle_to_fetch = create_and_save_vehicle_id(stub, evcc_config, vehicle)
            # vehicle_to_fetch = create_vehicle(evcc_config, stub)
            # log.debug("Vehicle client received: " + str(vehicle_to_fetch))
```
Selbe Anpassung auch im weiteren Code prüfen. Z.B. Zeilen 77/78

Reply:
Das hatte ich nicht angefasst, weil es original im Master so war.
Habe ich jetzt aber auch ordentlich formatiert.